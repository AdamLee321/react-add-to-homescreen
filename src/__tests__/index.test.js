/* global describe it expect */
import React from 'react';
import { shallow } from 'enzyme';

import AddToHomescreen from '../index.js';

describe('AddToHomescreen Component', () => {
  it('should render', done => {
    const wrapper = shallow(<AddToHomescreen />);
    expect(wrapper).toMatchSnapshot();

    done();
  });
});
