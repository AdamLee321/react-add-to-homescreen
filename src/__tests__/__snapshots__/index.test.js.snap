// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToHomescreen Component should render 1`] = `
<div>
  This is so!
</div>
`;
